llava/model/builder.py

```diff
@@ -95,7 +96,8 @@ def load_from_hf(repo_id, filename, subfolder=None):
                 model = LlavaMptForCausalLM.from_pretrained(model_base, low_cpu_mem_usage=True, config=cfg_pretrained, **kwargs)
             else:
                 tokenizer = AutoTokenizer.from_pretrained(model_base, use_fast=False)
-                cfg_pretrained = AutoConfig.from_pretrained(model_path)
+                from llava.model.language_model.llava_llama import LlavaConfig
```

Comment:
This is where our code should be reaching, but currently it does not because it doesnt have the term 'llava' in its model_name